src/frontend/src/pages/PIAActiveListPage/index.tsx

```diff
@@ -1,6 +1,17 @@
+import { AppStorage } from '../../utils/storage';
 import ListViewRender from '../../components/public/ListViewRender';
+import { TokenStorageKeys } from '../../utils/auth';
 
 const PIAActiveList = () => {
+  const freshLoginState = AppStorage.getItem(
+    TokenStorageKeys.FRESH_LOGIN_STATE,
+  );
+  const redirect = AppStorage.getItem(TokenStorageKeys.POST_LOGIN_REDIRECT_URL);
```

Comment:
Is there no issue with not seeing any definition of the some-origin check regarding the redirection?

For example,

```js
if (redirect.indexOf(window.location.origin) == 0) {
    window.location.href = redirect;
}
```

Reply:
I'm not sure what you mean by this